testing/iframe.js

```diff
@@ -276,16 +276,23 @@ export function createIframePromise(opt_runtimeOff, opt_beforeLayoutCallback) {
                       element.appendChild(placeholder);
                     }
                   }
+                  const resources = Services.resourcesForDoc(ampdoc);
+                  const resource = resources.getResourceForElementOptional(
+                    element
+                  );
+                  if (resource) {
```

Comment:
when won't there be a resource?

Reply:
Not a real thing per se. But happens in tests, when a tested element doesn't have a resource. TBH, I'd like to fix all those odd tests, but don't have time right now.

Reply:
Thank you for explaining! Agreed. So many oddities + impossible situations within tests